jsonrpsee/Cargo.toml

```diff
@@ -30,7 +30,7 @@ async-client = ["jsonrpsee-core/async-client"]
 http-client = ["jsonrpsee-http-client", "jsonrpsee-types", "jsonrpsee-core"]
 http-server = ["jsonrpsee-http-server", "jsonrpsee-types", "jsonrpsee-core"]
 wasm-client = ["jsonrpsee-wasm-client", "jsonrpsee-types", "jsonrpsee-core"]
-ws-client = ["jsonrpsee-ws-client", "jsonrpsee-types", "jsonrpsee-core"]
+ws-client = ["jsonrpsee-ws-client", "jsonrpsee-types", "jsonrpsee-core", "jsonrpsee-core/async-wasm-client"]
```

Comment:
hmm, this shouldn't be needed?

if needed it should be "jsonrpsee-core/async-client"